plenum/server/monitor.py

```diff
@@ -491,8 +549,8 @@ def getThroughput(self, instId: int) -> float:
         # node are started at almost the same time.
         if instId >= self.instances.count:
             return None
-        reqs, tm = self.numOrderedRequests[instId]
-        return reqs / tm if tm else 0
+        throughput = self.throughputs[instId].get_throughput()
+        return throughput or 0
```

Comment:
This looks a bit brittle. What if someone makes window size different on master and backups? Returning None would clearly indicate that we don't yet have a correct throughput measurement instead of masking it with zero. Besides None is already handled in `masterThroughputRatio`